### ISSUE TITLE
{% highlight %} does not respect disabling of syntax highlighting

### ISSUE THREAD

**Issue by tkornack** [cobalt-org/cobalt.rs#640]
The html generated from markdown is exceptionally clean, with the exception of the code blocks, which have a fixed styling like this:

```
<pre style="background-color:#2b303b;">
<span style="background-color:#2b303b;color:#c0c5ce;">ans = 42;
</span></pre>
```

It would be so much better if the code blocks emitted the `<code>` tag without the fixed color syling, which would then allow us to style it using css.

**Comment by epage** [comment]
So I'm seeing two bugs here

1. We are not documenting [support for disabling syntax highlighting](https://cobalt-org.github.io/docs/config/) (added in #407)
2. Disabling of highlighting is limited to markdown code fences and needs to be extended to liquid `{% highlight %}` blocks.  The workaround would be to instead use markdown code fences.

**Comment by epage** [comment]
Classifying this as a bug due to the inconsistent behavior.

**Comment by tkornack** [comment]
Oh, thanks!

```
syntax_highlight:
  enabled: false
```

works perfectly.

**Comment by rdimartino** [comment]
This was fixed in https://github.com/cobalt-org/cobalt.rs/pull/973 and released in v0.18.0
